eng/WpfArcadeSdk/tools/FolderPaths.targets

```diff
@@ -14,6 +14,10 @@
     <!-- Consume from $(WpfTestArcadeWpfSdkPath) -->
     <WpfTransportedCommonDir Condition="'$(WpfTransportedCommonDir)'=='' And Exists('$(WpfArcadeSdkRoot)..\..\src\Microsoft.DotNet.Wpf\src\Common\')">$(WpfArcadeSdkRoot)..\..\src\Microsoft.DotNet.Wpf\src\Common\</WpfTransportedCommonDir>
 
+    <WpfTransportedGraphicsDir Condition="'$(WpfTransportedGraphicsDir)'=='' And Exists('$(WpfArcadeSdkRoot)src\WpfGfx\')">$(WpfArcadeSdkRoot)src\WpfGfx\</WpfTransportedGraphicsDir>
+    <WpfTransportedGraphicsDir Condition="'$(WpfTransportedGraphicsDir)'=='' And Exists('$(WpfArcadeSdkRoot)..\..\src\Microsoft.DotNet.Wpf\src\WpfGfx\')">$(WpfArcadeSdkRoot)..\..\src\Microsoft.DotNet.Wpf\src\WpfGfx\</WpfTransportedGraphicsDir>
+
+    <WpfTransportedGraphicsPath>$(WpfTransportedGraphicsDir)</WpfTransportedGraphicsPath>
```

Comment:
Why do we need a Dir property and a Path property when they are the same value?

Reply:
We really don't; when I was building this I noticed that the local convention here was `YYYDir`, but the convention elsewhere tends to be `YYYPath`. I figured I'd just respect both and invent an alias.  😞

Reply:
Yeah I don't think it's a huge deal, was just making sure there wasn't something I was missing about it.